resources/apt/apt.dsc.resource.json

```
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/bundled/resource/manifest.json",
+    "type": "DSC.PackageManagement/Apt",
+    "description": "DSC resource to manage Apt packages",
+    "tags": [
+        "Linux",
+        "apt",
+        "PackageManagement"
+    ],
+    "version": "0.1.0",
+    "get": {
+        "executable": "apt.dsc.resource.sh",
+        "args": [
+            "get"
+        ],
+        "input": "env"
+    },
+    "set": {
+        "executable": "apt.dsc.resource.sh",
+        "args": [
+            "set"
+        ],
+        "input": "env",
+        "implementsPretest": true,
+	"handlesExist": true
+    },
+    "export": {
+        "executable": "apt.dsc.resource.sh",
+        "args": [
+            "export"
+        ],
+        "input": "env"
+    },
+    "exitCodes": {
+        "0": "Success",
+        "1": "Invalid parameter"
+    },
+    "schema": {
+        "embedded": {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "$id": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/resources/DSC/PackageManagement/apt/v0.1.0/schema.json",
+            "title": "Apt",
+            "description": "Managed packages using apt",
+            "type": "object",
+            "required": [
+                "packageName"
+            ],
+            "additionalProperties": false,
+            "properties": {
+                "packageName": {
+                    "type": "string",
+                    "title": "Package Name",
+                    "description": "Defines the name of the package to query or install"
+                },
+                "version": {
+                    "type": "string",
+                    "title": "Version",
+                    "description": "Defines the version of the package to install"
+                },
+                "Source": {
+                    "type": "string",
+                    "title": "Source",
+                    "description": "Defines the source of the package"
+                },
+                "_exist": {
```

Standard implementation guidance for well-known properties is to define them as a reference to the schema for that property:

```suggestion
                "_exist": {
                    "$ref": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/resource/properties/exist.json"
                }
```

For reference, see [DSC Resource _exist property schema][01].

[01]: https://learn.microsoft.com/powershell/dsc/reference/schemas/resource/properties/exist?view=dsc-3.0&preserve-view=true